fe/fe-core/src/main/java/org/apache/doris/external/jdbc/JdbcClient.java

```diff
@@ -524,7 +526,8 @@ public Type mysqlTypeToDoris(JdbcFieldSchema fieldSchema) {
             case "TIMESTAMP":
             case "DATETIME":
             case "DATETIMEV2": // for jdbc catalog connecting Doris database
-                return ScalarType.createDatetimeV2Type(0);
+                // mysql can support microsecond
```

Comment:
Add TODO here.
We should get real precision of DATETIMEV2.

Reply:
done